.github/workflows/ci.yml

```diff
@@ -21,3 +21,19 @@ jobs:
           node-version: 14
       - run: npm install
       - run: npm test
+  deploy:
+    runs-on: ubuntu-latest
+    needs: test
+    if: github.ref == 'refs/heads/main'
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 14
```

Comment:
Security support ends in 2 months and 3 weeks
(30 Apr 2023)

https://endoflife.date/nodejs

IIRC node 18 (latest LTS) worked for you in building node-gyp? That should be used (Support ends in 2 years and 2 months
(30 Apr 2025))

Reply:
So you suggest this?
```yaml
- uses: actions/checkout@v3
- uses: actions/setup-node@v3
  with:
    node-version: 18
```

I saw the deprecation warning, but it stated Node12 not Node14

Reply:
Yes. And I suggest changing the node version a few lines above as well.